packages/components/color-picker/src/use-color-slider.ts

```diff
@@ -355,6 +356,7 @@ export const useColorSlider = ({
         "aria-valuenow": value,
         "aria-valuemin": min,
         "aria-valuemax": max,
+        "aria-valuetext": ariaValueText ?? value.toString(),
```

Comment:
It seems that `ariaValueText` has the following characteristics, 
so it might be better to add more user-friendly wording.

https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaValueText
>The ariaValueText property of the [Element](https://developer.mozilla.org/en-US/docs/Web/API/Element) interface reflects the value of the [aria-valuetext](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuetext) attribute, which defines the human-readable text alternative of aria-valuenow for a range widget.

Reply:
Saturation slider might be helpful for reference.

https://github.com/yamada-ui/yamada-ui/blob/2512e87fc92204b7e3d59b10cbe1ac07ddf2dc2a/packages/components/color-picker/src/use-saturation-slider.ts#L372

Reply:
@setodeve 
Thank you for confirming.
I will add user friendly wording.

Reply:
@setodeve 
Since it seems to be common for alpha values to indicate transparency from 0% to 100% using “%” as the unit, how about adding a % unit to the value by multiplying value by 100?

https://developer.mozilla.org/ja/docs/Web/CSS/opacity#alpha-value
https://ja.wikipedia.org/wiki/%E3%82%A2%E3%83%AB%E3%83%95%E3%82%A1%E5%80%A4_(%E3%83%87%E3%82%B8%E3%82%BF%E3%83%AB%E7%94%BB%E5%83%8F)

Reply:
“aria-valuetext = ‘50%’” is a good pattern!